marquez_client/utils.py

```diff
@@ -92,3 +94,8 @@ def now_ms():
     def utc_now():
         return str(generate(datetime.utcnow().replace(tzinfo=pytz.utc),
                             microseconds=True))
+
+    @staticmethod
+    def log_details(details):
```

Comment:
Do we need a utils method for the log details? The http request will only be logged as json in `LogBackend`

Reply:
more than once, goes to utils.